mod/groups/views/default/resources/groups/invitations.php

```diff
@@ -1,29 +1,27 @@
 <?php
 
-$username = elgg_extract('username', $vars);
-if ($username) {
-	$user = get_user_by_username($username);
-} else {
-	$user = elgg_get_logged_in_user_entity();
-}
-
-if (!$user || !$user->canEdit()) {
-	throw new \Elgg\EntityPermissionsException();
-}
+$user = elgg_get_page_owner_entity();
 
 elgg_push_breadcrumb(elgg_echo('groups'), "groups/all");
 
-elgg_set_page_owner_guid($user->guid);
-
+// build page elements
 $title = elgg_echo('groups:invitations');
 
-$content = elgg_view('groups/invitationrequests');
-
-$params = [
-	'content' => $content,
+$content = elgg_call(ELGG_IGNORE_ACCESS, function() use ($user) {
+	return elgg_list_relationships([
+		'relationship' => 'invited',
+		'relationship_guid' => $user->guid,
+		'inverse_relationship' => true,
+		'no_results' => elgg_echo('groups:invitations:none'),
+	]);
+});
+
+// build page
+$body = elgg_view_layout('content', [
```

Comment:
default

Reply:
was already a `content` layout, not changing it now